bucket/nextcloud.json

```
@@ -0,0 +1,27 @@
+{
+    "version": "2.5.3",
+    "description": "Desktop sync client for Nextcloud, a self-hosted productivity platform that keeps you in control.",
+    "homepage": "https://nextcloud.com/",
+    "license": "GPL-2.0-or-later",
+    "url": "https://download.nextcloud.com/desktop/releases/Windows/Nextcloud-2.5.3-setup.exe#/dl.7z",
+    "hash": "c75f667e6a149f5f72db46d15b4ea24a614789a2cf103040c2f9a66fc3482466",
+    "bin": [
```

Keep defaults. You cannot be sure if application is not calling it statically.

```suggestion
    "bin": [
        "nextcloudcmd.exe",
````

---

How about

```
    "bin": [
        "nextcloudcmd.exe",
        [
            "nextcloudcmd.exe",
            "nextcloud"
        ]
    ],
```

or

```
    "bin": [
        "nextcloud.exe",
        [
            "nextcloudcmd.exe",
            "nextcloud"
        ]
    ],
```

---

My suggestion will make it this:

```json
"bin": [
    "cmd",
    [
         "cmd",
         "simple"
    ]
]
```

Both codes you provided are identical and same to what my suggestion is ...